DOCS: Edits to streamline Install OpenVINO Overview Page - port to 22.1 (#13868)
openvinotoolkit/openvino: docs/install_guides/installing-openvino-overview.md

**Python**

For developers working in Python, OpenVINO Development Tools can easily be installed using PyPI. See the <a href="openvino_docs_install_guides_install_dev_tools.html#python-developers">For Python Developers</a> section of the Install OpenVINO Development Tools page for instructions.

**C++**

For developers working in C++, the core OpenVINO Runtime libraries must be installed separately. Then, OpenVINO Development Tools can be installed using requirements files or PyPI. See the <a href="openvino_docs_install_guides_install_dev_tools.html#cpp-developers">For C++ Developers</a> section of the Install OpenVINO Development Tools page for instructions.

### Option 2. Install OpenVINO Runtime only

OpenVINO Runtime may also be installed on its own without OpenVINO Development Tools. This is recommended for users who already have an optimized model and want to deploy it in an application that uses OpenVINO for inference on their device. To install OpenVINO Runtime only, follow the instructions on the [Install OpenVINO Runtime](installing-openvino-runtime.md) page.

The following methods are available to install OpenVINO Runtime:

> * Linux: You can install OpenVINO Runtime using archive files or Docker. See [Install OpenVINO on Linux](installing-openvino-linux-header.md).

Reviewer: For 2022.1, we don't have the archive installation. Instead, we have Installer, APT, YUM, Conda. So this part can keep the original version.

Reviewer 2: ```suggestion
* Linux: You can install OpenVINO Runtime using an [Installer](installing-openvino-linux.md), [APT](installing-openvino-apt.md), [YUM](installing-openvino-yum.md), [Anaconda Cloud](installing-openvino-conda.md), or [Docker](installing-openvino-docker-linux.md).
```

> * Windows: You can install OpenVINO Runtime using archive files or Docker. See [Install OpenVINO on Windows](installing-openvino-windows-header.md).

Reviewer 2: ```suggestion
* Windows: You can install OpenVINO Runtime using an [Installer](installing-openvino-windows.md), [Anaconda Cloud](installing-openvino-conda.md), or [Docker](installing-openvino-docker-windows.md).
```

> * macOS: You can install OpenVINO Runtime using archive files or Docker. See [Install OpenVINO on macOS](installing-openvino-macos-header.md).

Reviewer 2: ```suggestion
* Windows: You can install OpenVINO Runtime using an [Installer](installing-openvino-windows.md), [Anaconda Cloud](installing-openvino-conda.md), or [Docker](installing-openvino-docker-windows.md).
```

* [Raspbian OS](installing-openvino-raspbian.md)

### Option 3. Build OpenVINO from source

Source files are also available in the OpenVINO Toolkit GitHub repository. If you want to build OpenVINO from source for your platform, follow the [OpenVINO Build Instructions](https://github.com/openvinotoolkit/openvino/wiki/BuildingCode).

## Next Steps
Still unsure if you want to install OpenVINO toolkit? Check out the [OpenVINO tutorials](../tutorials.md) to run example applications directly in your web browser without installing it locally. Here are some exciting demos you can explore:
- [Monodepth Estimation with OpenVINO](https://docs.openvino.ai/latest/notebooks/201-vision-monodepth-with-output.html)
- [Style Transfer on ONNX Models with OpenVINO](https://docs.openvino.ai/latest/notebooks/212-onnx-style-transfer-with-output.html)
- [OpenVINO API Tutorial](https://docs.openvino.ai/latest/notebooks/002-openvino-api-with-output.html)

Follow these links to install OpenVINO:
- [Install OpenVINO Development Tools](installing-model-dev-tools.md)
- [Install OpenVINO Runtime](installing-openvino-runtime.md)

> - [Build from Source](https://github.com/openvinotoolkit/openvino/wiki/BuildingCode)

Reviewer: Let's keep the yocto link here.

Reviewer 2: ```suggestion
- [Build from Source](https://github.com/openvinotoolkit/openvino/wiki/BuildingCode)
- [Create a Yocto Image](installing-openvino-yocto.md)
```
